### PR TITLE
Add IPNetSlice and unit tests

### PR DESCRIPTION
**Pull request by rpothier**
This adds slices for IPNet.
This is needed for IPv4, IPv6 dual stack in kubernetes.

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @eparis @spf13 ptal 🤗

File: ipnet_slice.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+	*ipnsv.value = val
+	return ipnsv
+}
+
+// Set converts, and assigns, the comma-separated IPNet argument string representation as the []net.IPNet value of this flag.
+// If Set is called on a flag that already has a []net.IPNet assigned, the newly converted values will be appended.
+func (s *ipNetSliceValue) Set(val string) error {
+
+	// remove all quote characters
+	rmQuote := strings.NewReplacer(`"`, "", `'`, "", "`", "")
```
[the remark this answers is not in the document]
Comment (rpothier) [reply ipnet_slice.go]: The unit tests are modeled after ip_slice tests, which one of the tests is to test bad quoting.
The quotes need to be removed to handle this case of funny quoting.
If this is not needed, I can remove it.

[the remark this answers is not in the document]
Comment (rpothier) [reply ipnet_slice.go]: @eparis I answered the question, please let me know if this is still a concern.

[the remark this answers is not in the document]
Comment (thaJeztah) [reply ipnet_slice.go]: ping @eparis ptal 😅

**Comment by rpothier** [comment]
@eparis If you have a chance, can you take another look at this?
This is needed for the Dual Stack feature in Kubernetes - https://github.com/kubernetes/community/pull/2254

**Comment by AkihiroSuda** [comment]
ping @spf13 @eparis

**Comment by thaJeztah** [comment]
ping @spf13 @eparis PTAL; is there anything outstanding to get this merged?

**Comment by Arvinderpal** [comment]
@thaJeztah @eparis can we merge this? I believe @rpothier is no longer working on the dual-stack work in kubernetes but this feature would be very helpful to others who are. Thank you.

**Comment by thaJeztah** [comment]
I'm not a maintainer in this repository, and don't have write access.

@spf13 any way your could help getting this moving?

**Comment by rpothier** [comment]
Yea, I've been reassigned and no longer on Kubernetes.
Still I hope this goes in.

**Comment by thaJeztah** [comment]
@therealmitchconnors @eparis PTAL 🙏

**Comment by thaJeztah** [comment]
Thanks!

**Pull request merged**